lib/common/header.ts

```diff
@@ -0,0 +1,28 @@
+import { color, stripColors } from "../color";
+
+export function printHeader() {
+	if (process.env.HIDE_HEADER) return;
+	const version = "8.5.3";
+	const middle = [
+		color.dim("│  "),
+		color.cyanBright.bold(" {N} NativeScript "),
+		color.whiteBright.bold("CLI"),
+		color.dim(` [v${version}] `),
+		color.dim("  │"),
+	].join("");
+	const middle2 = [
+		color.dim("│ "),
+		color.whiteBright.bold(" Empowering JS with Native APIs "),
```

Comment:
```suggestion
		color.whiteBright.bold(" Empower JavaScript with native APIs "),
```

This will match heading on homepage 1-1 which helps clarify in full.